packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -207,6 +213,11 @@ export class FileTreeWidget extends CompressedTreeWidget {
         return DirNode.getContainingDir(node);
     }
 
+    protected getDropEffect(event: React.DragEvent): 'copy' | 'move' {
+        const isCopy = (event.ctrlKey && !isOSX) || (event.altKey && isOSX);
```

Comment:
I'd probably do something like 
```suggestion
        const isCopy = isOSX ? event.altKey : event.ctrlKey;
```